README.md

```diff
@@ -94,6 +94,16 @@ Putting it all together, it is 6 variant test cases.
 - Applying the `Web variant` to a `Standard Web project`
 - Applying the `Web variant` to a `Custom Web project`
 
+### Release
+
+Set the `HEX_API_KEY` as a Github secret (skip this step if it has been done)
+
+The release process is following the [Git flow](https://nimblehq.co/compass/development/version-control/#releases-).
```

Comment:
Tense:

```suggestion
The release process follows the [Git flow](https://nimblehq.co/compass/development/version-control/#releases-).
```

There is no need for the continuous present here as the process has been defined and is no longer active.

Reply:
Fixed in https://github.com/nimblehq/elixir-templates/pull/63/commits/b6cc5fc1aaf57f9bb3d280bbc73ce105c357b8bf @olivierobert